test/web-platform-tests/to-upstream/html/editing/activation/click-event-properties-dontupstream.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>click() should do nothing on disabled elements</title>
+<link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#attr-button-type-submit-state">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<button>Click me</button>
+
+<script>
+"use strict";
+
+// these tests should not be upstreamed since:
+// 1. we're testing pageX/Y and scroll offsets by setting scrollX/Y which should be readonly; jsdom doesn't implement
+//    scroll() and friends (yet)
+// 2. getBoundingClientRect() doesn't know about layout/scrolling, so jsdom can return some unusual offsetX/Y values
+//    during dispatch despite the calculation logic being correct (see below)
+// 3. equivalent tests are being added upstream in https://github.com/web-platform-tests/wpt/pull/38912 that use scroll()
+//    and expect correct offsetX/Y values during dispatch
+test(() => {
+  window.scrollX = 10;
+  window.scrollY = 20;
```

Comment:
Oof, I didn't realize we were setting these in tests. I'd like to avoid writing tests that rely on invalid behavior like this. Ideally we should fix scrollX/scrollY to be readonly.